src/prefect/client/orchestration.py

```diff
@@ -2694,6 +2696,74 @@ async def release_concurrency_slots(
             },
         )
 
+    async def create_global_concurrency_limit(
+        self, concurrency_limit: GlobalConcurrencyLimitCreate
+    ) -> UUID:
+        response = await self._client.post(
+            "/v2/concurrency_limits/",
+            json=concurrency_limit.dict(json_compatible=True, exclude_unset=True),
+        )
+        return UUID(response.json()["id"])
+
+    async def update_global_concurrency_limit(
+        self, concurrency_limit: GlobalConcurrencyLimitUpdate
+    ) -> httpx.Response:
```

Comment:
I'm sorry to ask for yet another change here, but the more I think about this, the more I'm not sure that this is the actual interface we want to provide here. The `rename_to` attribute is a creative solution, but I think a more straightforward option is to just put the `name` as an argument to the function:

```suggestion
    async def update_global_concurrency_limit(
        self, name: str, concurrency_limit: GlobalConcurrencyLimitUpdate
    ) -> httpx.Response:
```

This is consistent with our other update methods that accept an `id` and the data object, and gives us the `name` to use for generating the URL as well as allowing a user to provide a new name in the data object.